views/userDetails.njk

```diff
@@ -96,21 +96,40 @@
                   </td>
                 {% endif %}
               {% else %}
-                <td class="govuk-table__cell">
+                {% if staff.accountStatus == 'EXPIRED' %}
+                  <td class="govuk-table__cell">
+                  {{ govukTag({
+                    text: 'Inactive account - ' + staff.accountStatus,
+                    classes: "govuk-tag--yellow",
```

Comment:
?? Not sure if this should say Inactive at all?? The Jira ticket doesn't explicitly say it